parsers/BR.py

```diff
@@ -1,191 +1,184 @@
-#!/usr/bin/env python3
-
 from collections import defaultdict
+from parsers.BD import GENERATION_MAPPING
```

Comment:
:-1: - unnecessary coupling to save 8 lines - or is this just a mistake as you have `GENERATION_MAPPINGE` below

Reply:
importing the Bangladeshi GENERATION_MAPPING seems to be a mistake